e2e_test/source_inline/kafka/consumer_group.slt

```diff
@@ -0,0 +1,108 @@
+# Note: control substitution on will force us to use "\\n" instead of "\n" in commands
+control substitution on
+
+# Note either `./risedev rpk` or `rpk` is ok here.
+# risedev-env contains env var RPK_BROKERS, which is read by rpk
+system ok
+rpk topic create test_consumer_group -p 3
+
+system ok
+cat <<EOF | rpk topic produce test_consumer_group -f "%p %v\\n" -p 0
+0 {"x":"a"}
+1 {"x":"b"}
+2 {"x":"c"}
+EOF
+
+statement ok
+CREATE SOURCE s(x varchar)
+WITH(
+    ${RISEDEV_KAFKA_WITH_OPTIONS_COMMON},
+    topic = 'test_consumer_group',
+    scan.startup.mode = 'earliest',
+) FORMAT PLAIN ENCODE JSON;
+
+statement ok
+CREATE MATERIALIZED VIEW mv AS SELECT * from s;
+
+query ?
+SELECT * FROM s order by x;
+----
+a
+b
+c
+
+sleep 2s
+
+query ?
+SELECT * FROM mv order by x;
+----
+a
+b
+c
+
+
+# There are 4 consumer groups, 1 for batch query (not listed below), 3 for MV.
+# All of them are "Empty" state with 0 members, because we manually `assign` partitions to them.
+# At the begginning, the MV's consumer group will not occur. They will be created after committing offset to Kafka.
+# (enable.auto.commit defaults to true, and auto.commit.interval.ms defaults to 5s)
+sleep 5s
+
+system ok
+./e2e_test/source_inline/kafka/consumer_group.mjs --mv mv list-members
```

Comment:
using js here is just a personal choice. It doesn't matter much. Can also use `sh`, `py`...

Here a script is needed and plain bash command is not enough because we need to first query fragment id, and then filter consumer groups with that.

Reply:
Can we automatically download/install `zx` for developers running tests locally, like through `cargo make`? Or, include this in the developer documentation.